bin/trackers/Tracker_Regex.py

```diff
@@ -93,7 +96,24 @@ def new_tracker_found(self, tracker, tracker_type, item):
             for mail in mail_to_notify:
                 NotificationHelper.sendEmailNotification(mail, mail_subject, mail_body)
 
-if __name__ == "__main__":
+            # Webhook
+            webhook_to_post = Term.get_term_webhook(tracker_uuid)
+            if webhook_to_post:
+                json_request = {"trackerId": tracker_uuid,
+                                "itemId": item_id,
+                                "itemURL": self.full_item_url + item_id,
+                                "tracker": tracker,
+                                "itemSource": item_source,
+                                "itemDate": item_date,
+                                "tags": tags_to_add,
+                                "emailNotification": f'{mail_to_notify}',
+                                "trackerType": tracker_type
+                                }
+                response = requests.post(webhook_to_post, json=json_request)
+                if response.status_code >= 400:
+                    raise Exception(f"Webhook request failed for {webhook_to_post}\nReason: {response.reason}")
```

Comment:
I don't think we should stop the module if we can't reach the webhook.
Can you replace this exception by an error message ?

You can use this function:
```python
self.redis_logger.error(error_message)
```